psi4/src/psi4/dfmp2/mp2.cc

```diff
@@ -2411,16 +2418,18 @@ void RDFMP2::form_Z()
         Dtemp->scale(0.5);
 
         // Add in the reference contribution
+
         for (int i = 0; i < nocc; ++i)
             Dtemp->add(i, i, 1.0);
-        Ca_ = SharedMatrix(new Matrix("DF-MP2 (unrelaxed) Natural Orbitals", nsopi_, nmopi_));
-        epsilon_a_ = SharedVector(new Vector("DF-MP2 (unrelaxed) NO Occupations", nmopi_));
+        Ca__ = SharedMatrix(new Matrix("DF-MP2 (unrelaxed) Natural Orbitals", nsopi_, nmopi_));
+        epsilon_a__ = SharedVector(new Vector("DF-MP2 (unrelaxed) NO Occupations", nmopi_));
```

Comment:
I see. However, its confusing when one uses the fchk_interface as `Ca_` and `epsilon_a_` which are natural orbitals and occupation numbers get printed as `Alpha MO coefficients` and `Alpha Orbital Energies` respectively in the checkpoint file. Do you think we should provide the original MO coefficients and orbital energies of the reference wavefunction as well in the df-mp2 gradient case since all the other modules do the same?